library/StateControl.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import json
+
+from library.common import *
+from library.ImageControl import ImageControl
+from library.PreseedControl import PreseedControl
+
+class StateControl(object):
+    def __init__(self, requester, machine_id):
+        self.requester = requester
+        self.machine_id = machine_id
+
+    def machine_provision(self):
```

Comment:
Shouldn't this method be called "PXE boot"?

Reply:
I'm sticking to the MrP terminology, the state is called 'provision' so it makes more sense calling it that.

Reply:
I'm sticking to MrP terminology in which the provisioning machine state is 'provision'. PXE is a 'power' state.

Reply:
I see. Makes sense.